assets/section-image-banner.css

```diff
@@ -317,6 +317,7 @@
 
 @media screen and (min-width: 750px) {
   .banner--desktop-transparent .banner__box {
+    background: transparent;
```

Comment:
I also checked the Image banner section and its working great!

Would there be any downsides to replacing `background-color: transparent;` with `background: transparent;` vs declaring both? I tested this and although it works, I'm wondering if there's a scenario where both are needed 🤔

Reply:
Yeah you're right. I can probably just keep the one 🤔